update-stable-ids/src/main/java/org/reactome/release/updateStableIds/Main.java

```diff
@@ -36,7 +36,7 @@ public static void main( String[] args ) throws Exception
        int releaseNumber = Integer.valueOf(props.getProperty("releaseNumber"));
        int prevReleaseNumber = releaseNumber - 1;
        
-       logger.info("Creating DB adaptors for test_slice_" + releaseNumber + ", test_slice_" + prevReleaseNumber + "_snapshot and gk_central");
+       logger.info("Creating DB adaptors");
```

Comment:
This would be a good log statement to include in a small private method that wraps the logic of extracting the db connection info from the properties file and then calling `new MySQLAdaptor`:

```
private createMySQLAdaptor(Properties props, String userProperty, String passwordProperty, String hostProperty, String databaseNameProperty, int portProperty) throw SQLException {
    String userName = props.getProperty(useProperty);
    String password = props.getProperty(passwordProperty);
    String host = props.getProperty(hostProperty);
    String databaseName = props.getProperty(databaseNameProperty);
    int port = Integer.parseInt(props.getProperty(portProperty));
    
    logger.info(
        "Creating DB adaptor for " + databaseName + " at " + host + " on port " + port + " for user " 
        + userName
    );

    return new MySQLAdaptor(host, databaseName, userName, password, port);
}
```